extensions/kusto/package.json

```diff
@@ -291,6 +291,10 @@
             {
               "displayName": "%kusto.connectionOptions.authType.categoryValues.azureMFA%",
               "name": "AzureMFA"
+            },
+            {
+              "displayName": "%kusto.connectionOptions.authType.categoryValues.dstsAuth%",
+              "name": "dstsAuth"
```

Comment:
This needs to live somewhere else but for now it's required for the option to be available in the Connection Widget

Reply:
This is because dstsAuth is not a generic kusto thing but rather specifically for livesite correct?

Reply:
Right. It's an internal authentication method used for Livesite.